opm/autodiff/RedistributeDataHandles.hpp

```diff
@@ -219,7 +219,7 @@ class BlackoilStateDataHandle
     {
         if ( T::codimension == 0)
         {
-            return 2 * sendState_.numPhases() +4+2*sendGrid_.numCellFaces(e.index());
+            return 2 * sendState_.numPhases() + 5 + 2*sendGrid_.numCellFaces(e.index());
```

Comment:
For the future (I do not require a change in this PR): these explicit numbers make me nervous, and I'd like for it to be less fragile. I think the best way here would be to add some tests that would fail if this was not updated properly.